CMakeLists.txt

```diff
@@ -92,8 +92,8 @@ if(NOT MAKE_DOCS_ONLY)
     add_subdirectory(${HPCC_SOURCE_DIR} oss EXCLUDE_FROM_ALL)
     message(STATUS "End setup of HPCC-Platform build dependency")
 
-    HPCC_add_subdirectory(esp)
-    HPCC_add_subdirectory(initfiles)
+    HPCC_ADD_SUBDIRECTORY(esp)
+    HPCC_ADD_SUBDIRECTORY(initfiles)
```

Comment:
That's a custom macro we created.  Kept it all caps to distinguish it from cmake functions.